<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>342</width>
    <height>227</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>342</width>
    <height>227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label_acct">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>76</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_passwd">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>112</y>
     <width>77</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_acct">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>76</y>
     <width>149</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>112</y>
     <width>149</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_save_passwd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>177</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>save name &amp;&amp; passwd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>170</y>
     <width>113</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
